conan-recipes/prebuilt/all/conandata.yml

```diff
@@ -0,0 +1,29 @@
+sources:
+  "0.10.1-rc":
+    "Windows":
+      "x86_64":
+        url: "https://github.com/eclipse-zenoh/zenoh-c/releases/download/0.10.1-rc/zenoh-c-0.10.1-rc-x86_64-pc-windows-msvc.zip"
+        sha256: "a59a4c33a160298409649f16bee5f32f14de52c03273fe29d4497934cf05789e"
+    "Linux":
+      "x86_64":
+        url: "https://github.com/eclipse-zenoh/zenoh-c/releases/download/0.10.1-rc/zenoh-c-0.10.1-rc-x86_64-unknown-linux-gnu.zip"
+        sha256: "0554b0fe753df3c023b09e005b309d654b1cc33ddc34190b5a5c370b72281c3b"
```

Comment:
I've been thinking on this for a few days to figure out how to deal with tagged releases and Conan packages on a few of the uProtocol libraries I'm working on, and have a question:

What is the workflow for making a tagged release? It looks like a tag would first need to be added and all of the binaries built / published. Then a follow-up commit _after_ the tagged commit would update the recipes so the Conan packages can be built from the new release. Is that correct?

That would mean that the tagged commit is not actually 100% the release because the recipes are wrong. But the tag can't be moved after the recipes are updated (maybe only in the build from source case?) without changing some of the hashes.

I'm sure I'm just missing a step somewhere 😅

Reply:
You are right, there is a bootstrap-like problem. It's something we have discussed internally and so far moving the tag and updating the hashes seems to be the way to go. This will be done as part of the release process.